libtransmission/jsonsl.c

```diff
@@ -792,7 +792,7 @@ populate_component(char *in,
     }
 
     /* Replace the next / with a NULL */
-    *next = strstr(in, "/");
+    *next = strchr(in, '/');
```

Comment:
```suggestion
    *next = strstr(in, "/");
```

Reply:
@ckerr fix it here https://github.com/transmission/transmission/pull/4172/commits/c9aae7fc36ca9b235674b36ac4ef69951ca653b5